.github/workflows/run-tests.yml

```diff
@@ -0,0 +1,41 @@
+name: Run tests
+
+# todo: stop workflow here if '#skip-e2e' was provided in $GIT_COMMIT_MESSAGE
+
+on:
+  push:
+    branches:
+      - '*'
+      - '*/*/*'
+      - '!master'
+      - '!master-shelley'
+      - '!develop'
+
+jobs:
+  build:
+
+    strategy:
+      fail-fast: false # don't fail Chrome if Firefox fails, etc.
+      matrix:
+        include:
+          - platform: chrome
+            container: circleci/node:12-browsers
```

Comment:
Fails with `An image label with the label circleci/node:12-browsers does not exist.`

According to the docs

```
CircleCI provides a set of pre-built images with common dependencies. These images have the USER set to circleci, which causes permissions to conflict with GitHub Actions.

We recommend that you move away from CircleCI's pre-built images when you migrate to GitHub Actions. In many cases, you can use actions to install the additional dependencies you need.
```